ecommerce/baskets/serializers.py

```diff
@@ -0,0 +1,68 @@
+from django.db.transaction import atomic
+from rest_framework import serializers
+
+from baskets.models import Basket, BasketItem
+from customers.serializers import CustomerSerializer
+from products.serializers import ProductSerializer, ProductDetailedSerializer
+
+
+class BasketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Basket
+        fields = ("id", "customer", "status")
+
+
+class BasketItemSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = BasketItem
+        fields = ("id", "basket", "product", "quantity", "price")
+
+
+class BasketDetailedSerializer(serializers.ModelSerializer):
+    customer = CustomerSerializer(many=False)
+
+    class Meta:
+        model = Basket
+        fields = ("customer", "status")
+
```

Comment:
In detailed serializers, maybe it's better to inherit from  the non-detailed serializer. For example, here in BasketDetailedSerializer, if inherit from BasketSerializer you don't need to repeat class Meta.